pyramid_oereb/standard/hook_methods.py

```diff
@@ -14,6 +15,9 @@
 from pyramid_oereb.lib.records.office import OfficeRecord
 
 
+log = logging.getLogger(__name__)
```

Comment:
you add it but it's not used

Reply:
it is not the same as (use lamda function)
`real_estate.public_law_restrictions = sorted(real_estate.public_law_restrictions, key=lambda x: x.get('type_code', ''))` (whithout the whole `cmp` function ?)

In all cases, with this code and these names: `['Arboretum', 'Ärboretum', 'Belize']` you will get this result:
``['Arboretum', 'Belize', 'Ärboretum']``
Because Ä > Z. It is the wanted result ?

Reply:
removed